src/TLuaInterpreter.cpp

```diff
@@ -12186,9 +12186,9 @@ std::pair<bool, bool> TLuaInterpreter::callLuaFunctionReturnBool(void* pT)
         lua_pop(L, lua_gettop(L));
         //lua_settop(L, 0);
         if (error == 0) {
-            return std::make_pair(true, returnValue);
```

Comment:
When we did this we didn't pick up on the redundant `== 0` - swapping the branches over we can avoid that slightly smelly construct.